src/remix/sdk-setup.ts

```diff
@@ -105,6 +105,8 @@ function insertServerInitCall(
   const initCall = builders.functionCall('Sentry.init', {
     dsn,
     tracesSampleRate: 1.0,
+    // uncomment the line below to enable Spotlight (https://spotlightjs.com)
+    // spotlight: process.env.NODE_ENV === 'development',
```

Comment:
Have you tested this in a Remix application? I don't think this works as you'd expect. Unless I'm missing something, this is a pure code comment and won't be picked up by the function call builder that generates the AST for the Sentry.init call. 

It's very closely related to what I wrote in #546 about AST manipulation. Maybe we need to find a way to make this work rather sooner than later. 

What I was trying to do earlier when reviewing #546 was using a recast builder to manually build the AST. It allows me to insert comments into an AST node like the options object. If you wanna take a look at this, we already do stuff like this in `ast-utils.ts`: https://github.com/Shubhdeep12/sentry-wizard/blob/088ffb2452d0a7a00521282a3f5de59d53d2d832/src/utils/ast-utils.ts#L121-L152

Maybe we can use this here, too. So basically change the magiacst builder to a `recast.types.builder` and manually construct things. But we have to find a way to preserve code formatting (as well as possible) when the code is generated.

Reply:
Got it, looks like magicast builder is removing the comments. my bad, I think I missed checking this. 
let me try the suggested sol.

Reply:
Hi @Lms24 not sure if we can add comments in objects in recast too using builder functions. 
But with magicast and recast, there is this way to add comments viz. writing code itself instead of the builder function.
I tried to put the code directly and then parsed it. It's working fine and not removing the comments now.

Any input here? Can we do this way?
![Screenshot from 2024-03-18 09-03-55](https://github.com/getsentry/sentry-wizard/assets/43654389/2e581898-e999-4065-9438-111db569c868)

Reply:
Hey @Shubhdeep12 yup, if that gets the job done, let's do it 👍

Reply:
pushed the fix @Lms24